server/http.go

```diff
@@ -435,26 +443,26 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 	post.Props["meeting_status"] = zoom.WebhookStatusEnded
 	post.Props["attachments"] = []*model.SlackAttachment{&slackAttachment}
 
-	_, appErr = p.API.UpdatePost(post)
-	if appErr != nil {
-		p.API.LogWarn("Could not update the post", "Error", appErr.Error())
+	if _, appErr = p.API.UpdatePost(post); appErr != nil {
+		p.API.LogWarn("Could not update the post", "err", appErr.Error())
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
 	}
 
 	if appErr = p.deleteMeetingPostID(meetingPostID); appErr != nil {
-		p.API.LogWarn("failed to delete db entry", "Error", appErr.Error())
+		p.API.LogWarn("failed to delete db entry", "error", appErr.Error())
 		return
 	}
 
 	w.Header().Set("Content-Type", "application/json")
 	if err := json.NewEncoder(w).Encode(post); err != nil {
+		p.API.LogWarn("failed to write response", "error", err.Error())
 		http.Error(w, "failed to write the response", http.StatusInternalServerError)
```

Comment:
If you are keeping this line here, why did you remove it above?

Reply:
Now removed this from here as well.
https://github.com/mattermost/mattermost-plugin-zoom/pull/284/files/f6c187931258d07dbe04cdfbbb883e94f866ba1b#r1061019522